### PR TITLE
Let absolute paths be used for test runner executable

### PR DESCRIPTION
**Pull request by subatta**

### CODE REVIEW

File: lib/albacore/task_types/test_runner.rb [hunk 1]
```diff
@@ -191,7 +191,7 @@ def handle_directory dll, exe, &block
                 [Paths.normalise_slashes(Pathname.new(File.dirname(dll))), Pathname.new(exe)]
             end
 
-          exe_rel = exe.relative_path_from dir
+          exe_rel = exe.to_s.include?(':') ? exe : exe.relative_path_from(dir)
```
Comment (haf) [inline lib/albacore/task_types/test_runner.rb]: What does the `:` mean?

Comment (subatta) [inline lib/albacore/task_types/test_runner.rb]: It's the colon in path after drive letter.

Comment (haf) [inline lib/albacore/task_types/test_runner.rb]: It's a path separator on linux.

Comment (subatta) [inline lib/albacore/task_types/test_runner.rb]: Yes, the issue was on windows and addresses that case only.

Comment (haf) [inline lib/albacore/task_types/test_runner.rb]: Wouldn't it be better to check if the path is rooted?

Comment (subatta) [inline lib/albacore/task_types/test_runner.rb]: Would something like this work? Don't have linux/unix to test: File.expand_path(path) == path

Comment (haf) [inline lib/albacore/task_types/test_runner.rb]: Why not http://stackoverflow.com/a/2316795/63621?

Comment (subatta) [inline lib/albacore/task_types/test_runner.rb]: Either one, I prefer expand_path, doesn't need to create an object.  Let me know which we want to end up with, I'll update PR.

Comment (haf) [inline lib/albacore/task_types/test_runner.rb]: Yes, but expand_path assumes that path separators are the same and will return false negatives cross-platform.

Comment (subatta) [inline lib/albacore/task_types/test_runner.rb]: Changed to `(Pathname.new exe.to_s).absolute` and updated PR.

**Comment by haf** [comment]
Ok, so does this tie into an issue on the tracker?

**Comment by subatta** [comment]
Added #224 after seeing your comment.

**Comment by haf** [comment]
Ok, thank you!

**Comment by haf** [comment]
I've removed test_runner from albacore because this PR or one before it broke the tests. I accept a PR to re-add it if whoever needs it makes the tests pass on OS X again.

**Pull request merged**